src/sh/keptn/Keptn.groovy

```diff
@@ -780,7 +797,8 @@ def sendFinishedEvent(Map args) {
 
     // load labels from args (if set)
     def labels = args.containsKey('labels') ? args.labels : [:]
-
+    def eventPayload = args.containsKey('eventPayload') ? args.eventPayload : [:]
```

Comment:
Question: Should we call it `eventTypePayload` instead of `eventPayload` for consistency?